react/ProductKit.js

```diff
@@ -85,40 +127,51 @@ class ProductKit extends Component {
     } = this.props
 
     if (!product.benefits.length) {
-      return ( <Fragment></Fragment> )
+      return ( 
+        <Fragment></Fragment>
+      )
     }
 
-    const { benefits: [ benefit] } = product
-    const kitItems = benefit.items
+    const { benefits: [ benefit ] } = product
+    
+    const kitProducts = benefit.items.slice(0, MAX_ITEMS).map(item => (
```

Comment:
When the function has the same parameters as the callback called by `map`, you can pass the reference to the function directly, e.g. `array.map(this.prepareProduct)`